domains/lasthub.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "LastHubHosting",
+           "email": "contactlasthub@gmail.com",
+           "discord": "925052103746027550"
+        },
+    
+        "record": {
+            "A": ["13.235.134.36"]
```

Comment:
Please give a preview of your website.
Also are you using this for your hosting service? If so we don't allow that.